deploy/template/etc/init/docker-worker.conf

```diff
@@ -18,11 +18,14 @@ script
     . /etc/default/$UPSTART_JOB
   fi
 
-  # Pull images used for sidecar containers
-  docker pull taskcluster/taskcluster-proxy:latest
-  docker pull taskcluster/livelog:v4
-  docker pull taskcluster/dind-service:v4.0
-  docker pull taskcluster/relengapi-proxy:2.0.1
+  echo "Loading docker sidecar images"
```

Comment:
I'm confused how this will change things for the user: if we were already docker-pulling these at startup, why does it help to docker load them instead?